test/runFIT.sh

```diff
@@ -59,6 +60,36 @@ collectTestReport()
     fi
 }
 
+########################################################
+#  Wait for virtual nodes to be discovered.
+#  This expects exactly the number of virtual compute nodes defined by nodes_count 
+########################################################
+waitForNodes() {
+    timeout=0
+    maxto=60
+    nodes_count=$1
+    rackhd_host=$2
+    rackhd_http_port=$3
+    set +e
+    url=http://$rackhd_host:$rackhd_http_port/api/2.0/nodes
+    # check if nodeids have been created for the virtual nodes
+    sleep 20
+    while [ ${timeout} != ${maxto} ]; do
+      echo "Current node list: "
+      wget -SO- -T 1 -t 1 --retry-connrefused --waitretry=1 --read-timeout=20 --timeout=15 --continue ${url}
```

Comment:
I knew it's copy-paste code, but curious why run this twice...

Reply:
Sorry, I don't know.